package.json

```diff
@@ -190,6 +191,7 @@
     "clsx": "^1.0.4",
     "comlink": "^4.3.0",
     "core-js": "^3.6.0",
+    "crypto-js": "^4.1.1",
```

Comment:
right, but as @nev-r pointed out without any type of authorization a local application can execute actions on DIM without the user knows (ofc if the stream deck feature is already enabled). It's a unlikely situation but for example an application can expose a WebSocket server on the same port before the Stream Deck application (sure can't be nothing dangerous, but something to troll or capture data maybe ?).

Reply:
I think your "look for the number" authorization scheme is enough to prevent that. Encrypting the channel doesn't seem necessary, unless there's something I'm not thinking about where something takes over the socket without it closing?